web/pageComponents/pageTemplates/TopicPage.tsx

```diff
@@ -34,7 +34,7 @@ type TopicPageProps = {
 const TopicPage = ({ data }: TopicPageProps) => {
   const titleStyles = useSharedTitleStyles(data?.hero?.type, data?.content?.[0])
   const { breadcrumbs } = data
-
+  data.hero.background = data.hero.type !== HeroTypes.DEFAULT ? titleStyles.backgroundColor : data.hero.background
```

Comment:
This solution is changing the background of all banners, that is not what we want to achieve. It erroneously changes the background color of a 50/50 banner to match the next section for example.

To achieve what you want you can add a new prop to the SharedBanner and pass it down to the full-width banner:

`<SharedBanner title={data.title} hero={data.hero} captionBg={titleStyles.bgColor} />
`

and 

```
.
.
.
export const SharedBanner = ({ title, hero, hideImageCaption, captionBg }: BannerProps) => {
  switch (hero.type) {
    case HeroTypes.FULL_WIDTH_IMAGE:
      return (
        <FullImageHero
          ratio={hero.ratio as string}
          figure={hero.figure}
          hideImageCaption={hideImageCaption}
          captionBg={captionBg} // <---- THIS
        />
      )
.
.
.
```